index.js

```diff
@@ -4,7 +4,39 @@ var _ = require('lodash');
 var Prober = require('airlock');
 var request = require('request');
 
-module.exports.ClustoClient = ClustoClient;
+var CLUSTO_TYPES = [
+    'appliance',
+    'cage',
+    'clustometa',
+    'consoleserver',
+    'datacenter',
+    'generic',
+    'location',
+    'networkswitch',
+    'pipeline',
+    'pod',
+    'pool',
+    'powerstrip',
+    'rack',
+    'resourcemanager',
+    'role',
+    'server',
+    'zone'
+];
+var CLUSTO_TYPE_PLURALS = {
+    'networkswitch': 'networkswitches'
+};
+
+module.exports = {
+    CLUSTO_TYPES: CLUSTO_TYPES,
+    CLUSTO_TYPE_PLURALS: CLUSTO_TYPE_PLURALS,
+    ClustoClient: ClustoClient,
+    pluralize: pluralize
+};
+
+function pluralize(word, dictionary) {
+    return _.get(dictionary, word, word + 's');
+}
```

Comment:
Could you move pluralize into a `util.js` to avoid exporting it as the public interface of the module perhaps?